src/NexusMods.App/Program.cs

```diff
@@ -111,23 +111,18 @@ private static AppConfig ReadAppConfig(AppConfig existingConfig)
         const string configFileName = "AppConfig.json";
 
         // TODO: NexusMods.Paths needs ReadAllText API. For now we delegate to standard library because source is `FileSystem.Shared`.
-        var appFolder = FileSystem.Shared.GetKnownPath(KnownPath.EntryDirectory);
-        try
-        {
-            return File.ReadAllText(appFolder.Combine(configFileName).GetFullPath());
-        }
-        catch (Exception)
+
+        var fs = FileSystem.Shared;
+        if (fs.OS.IsLinux)
         {
-            // Config doesn't exist.
-            // Try the OWD environment variable in case this is an AppImage
+            // On AppImage (Linux), 'OWD' should take precedence over the entry directory if it exists.
             var owd = Environment.GetEnvironmentVariable("OWD");
```

Comment:
Add a link to https://docs.appimage.org/packaging-guide/environment-variables.html.